android/keyple-plugin/android-omapi/src/main/kotlin/org/eclipse/keyple/plugin/android/omapi/AbstractAndroidOmapiPlugin.kt

```diff
@@ -57,8 +57,6 @@ internal abstract class AbstractAndroidOmapiPlugin<T, V> : AbstractPlugin(PLUGIN
 
         if (readers.isEmpty()) {
             Timber.w("OMAPI SeService is not connected yet")
-            // throw new KeypleReaderException("OMAPI SeService is not connected yet, try again");
```

Comment:
Expected Behaviour: raise and exception and stop process or return empty reader list?